src/Sulu/Component/Content/Document/Subscriber/StructureTypeFilingSubscriber.php

```diff
@@ -42,7 +43,7 @@ protected function generatePath(PersistEvent $event)
         if ($event->hasParentNode()) {
             $currentPath = $event->getParentNode()->getPath();
         }
-        $parentName = $this->getParentName($document);
+        $parentName = $this->getParentName($event);
```

Comment:
Why not passing `$event->getDocument()` here? It's safer to only pass what the function needs.